mirgecom/eos.py

```diff
@@ -0,0 +1,104 @@
+__copyright__ = (
+    "Copyright (C) 2020 University of Illinois Board of Trustees"
+)
```

Comment:
Include authorship info

Reply:
__author__ = """                                                                                                
Center for Exascale-Enabled Scramjet Design                                                                     
University of Illinois, Urbana, IL 61801                                                                        
"""

Reply:
Not what I was thinking of. What I was thinking was more akin to a manually maintained `git blame`. But since we have `git blame`, there's really no reason for that, so nvm. :)

Side point: Triple backquotes in Markdown to include code, like so:
```python
__author__ = """                                                                                                
Center for Exascale-Enabled Scramjet Design                                                                     
University of Illinois, Urbana, IL 61801                                                                        
"""
```